kliff/uq/mcmc.py

```diff
@@ -106,6 +106,11 @@ def __new__(
         **kwargs,
```

Comment:
How about change `use_ptsampler` to `sampler`, which accepts a string as argument, that default to `pt_sampler`? Then if a user provides `ensemble_sampler`, it selects the other one. 

By doing this, we can easily extend the code for other samplers maybe in the future. We can even accepting an instance of a sampler class instead of a string.

Reply:
I like this suggestion. Please see my updates in bbc37d1. I also updated the example and documentations, but I haven't included the discussion about parallelization.